indico/modules/events/papers/templates/management/overview.html

```diff
@@ -77,6 +104,30 @@
                 {% endif %}
             </div>
         </div>
+        <div class="section">
+            <div class="icon icon-hammer"></div>
+            <div class="text">
+                <div class="label">{% trans %}Paper judging{% endtrans %}</div>
+                {% set deadline = cfp.judge_deadline %}
+                {% if deadline %}
+                    {% trans date=deadline|format_date(timezone=tz), time=deadline|format_time(timezone=tz) -%}
+                        The paper judging ends on <strong>{{ date }}</strong> at <strong>{{ time }}</strong>.
+                    {%- endtrans %}
+                {% else %}
+                    {%- trans %}The deadline for paper judging has not been set.{% endtrans -%}
```

Comment:
"There is no deadline for paper judging."? Otherwise it sounds like it's something you are supposed to change